docs/_docs/monitoring-metrics/new-metrics.adoc

```diff
@@ -320,15 +326,33 @@ Register name: `io.datastorage`
 [cols="2,1,3",opts="header"]
 |===
 |Name |    Type |    Description
+|CheckpointBeforeLockHistogram| histogram |   Histogram of checkpoint action before taken write lock durations in milliseconds.
+|CheckpointFsyncHistogram| histogram |   Histogram of checkpoint fsync durations in milliseconds.
+|CheckpointHistogram| histogram |   Histogram of checkpoint durations in milliseconds.
+|CheckpointListenersExecuteHistogram| histogram |   Histogram of checkpoint execution listeners under write lock durations in milliseconds.
+|CheckpointLockHoldHistogram| histogram |   Histogram of checkpoint lock hold durations in milliseconds.
+|CheckpointLockWaitHistogram| histogram |   Histogram of checkpoint lock wait durations in milliseconds.
+|CheckpointMarkHistogram| histogram |   Histogram of checkpoint mark durations in milliseconds.
+|CheckpointPagesWriteHistogram| histogram |   Histogram of checkpoint pages write durations in milliseconds.
+|CheckpointSplitAndSortPagesHistogram| histogram |   Histogram of splitting and sorting checkpoint pages durations in milliseconds.
 |CheckpointTotalTime| long |   Total duration of checkpoint
+|CheckpointWalRecordFsyncHistogram| histogram |   Histogram of the WAL fsync after logging ChTotalNodeseckpointRecord on begin of checkpoint durations in milliseconds.
+|CheckpointWriteEntryHistogram| histogram |   Histogram of entry buffer writing to file durations in milliseconds.
```

Comment:
Are you sure that it is "duration**s**" (plural), not "duration" (singular) here?

Reply:
@Nikita-tech-writer , It seems that singular is correct. I have fixed docs and metrics descriptions.